### PR TITLE
Add second QPS axis to multichart

### LINKED ISSUE #211: graph QPS on secondary Y axis
for multi graphs

### PR DESCRIPTION
**Pull request by Tahler**
Resolves #211.

image

### CODE REVIEW

File: ui/static/js/fortio_chart.js [hunk 1]
```diff
@@ -372,7 +373,7 @@ function deleteSingleChart () {
   chart = {}
 }
 
-function makeMultiChart (data) {
```
Comment (ldemailly) [inline ui/static/js/fortio_chart.js:375]: I know data happens to also be global but isn't it cleaner to pass it anyway ?

**Comment by ldemailly** [comment]
thx for this ! see comment/question

File: ui/static/js/fortio_chart.js (hunk 1 above)
Comment (Tahler) [reply ui/static/js/fortio_chart.js:375]: `data` is not used in `makeMultiChart`. The data for the chart is added over time with `fortioAddToMultiResult`

Comment (ldemailly) [reply ui/static/js/fortio_chart.js:375]: oh ! :( my bad ! ok some day we should objectify this whole thing but separately from adding features

**Review by ldemailly — APPROVED** [review]

**Pull request merged**